src/app/src/AboutSlide.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const Slide = styled.div`
+    height: 80vh;
+    padding: 1rem;
+    display: flex;
+
+    .info {
+        width: 40%;
+        padding: 2rem;
+        padding-top: 30%;
+    }
+
+    .video-container {
+        width: 60%;
+        display: flex;
+
+        video {
+            margin: auto;
+            height: fit-content;
+            max-width: 100%;
+        }
+
+        .play-button {
+            position: absolute;
+            background: gray;
+            height: 300px;
+            width: 300px;
+            border-radius: 50%;
+            opacity: 0.5;
+            top: 50%;
+            left: 18%;
+            transform: translateY(-50%);
+            font-size: 150px;
+            padding-top: 60px;
+            text-align: center;
+        }
+    }
```

Comment:
You _could_ argue pushing this off since #51 is adding a styled-components base library. That said, I think there's some componentization that would exist still outside the Rebass framework.